src/gui/tray/activitydata.cpp

```diff
@@ -15,22 +15,112 @@
 #include <QtCore>
 
 #include "activitydata.h"
+#include "folderman.h"
 
 
 namespace OCC {
 
-bool operator<(const Activity &rhs, const Activity &lhs)
-{
-    return rhs._dateTime > lhs._dateTime;
-}
+    bool operator<(const Activity &rhs, const Activity &lhs)
+    {
+        return rhs._dateTime > lhs._dateTime;
+    }
 
-bool operator==(const Activity &rhs, const Activity &lhs)
-{
-    return (rhs._type == lhs._type && rhs._id == lhs._id && rhs._accName == lhs._accName);
-}
+    bool operator==(const Activity &rhs, const Activity &lhs)
+    {
+        return (rhs._type == lhs._type && rhs._id == lhs._id && rhs._accName == lhs._accName);
+    }
```

Comment:
we do not add indentation inside a namespace
you have extra spaces

Reply:
Got it, removed